### PR TITLE
feat: adding members to organisation

### PR DESCRIPTION
**Pull request by MuthoniMN**
## What does this PR do?
This PR creates an endpoint that allows organisation owners to add new members

## How to Test
1. Create an organisation using POST `/api/v1/organisations`
2. Create a new user and get the user ID
3. Add user to the organisation using POST `/api/v1/organisations/:org_id/users`

## Edge Cases
- Test whether the organisation exists
- Test whether the current user is the owner of the organisation
- Test whether the specified user exists
- Test whether the specified user is already a member of the organisation

## Checklist
- [x] creating the `/api/v1/organisations/:org_id/users`
- [x] validating the org_id parameter
- [x] checking whether the current user is the owner of the organisation
- [x] validating the user_id parameter
- [x] checking whether the specified user is already a member
- [x] writing tests to validate parameters and to test edge cases

### Successful Response
```
{
        status: 'Success',
        message: 'User added successfully',
        status_code: 200,
}
```
### Error Response when the organisation does not exist
```
{
        status: 'error',
        message: 'Organisation not found',
        status_code: 404,
}
```
### Error Response when the current user is not the owner of the organisation
```
{
         status: 'Forbidden',
          message: 'Only admins can add users',
          status_code: 403,
}
```
### Error Response when the user does not exist
```
{
       status: 'error',
        message: 'User not found',
        status_code: 404,
}
```
### Error Response when the user is already a member of the organisation
```
{
          status: 'error',
          message: 'User already added to organization.',
          status_code: 409,
}
```

## Associated Issue
Issue #274

### CODE REVIEW

File: src/modules/organisations/organisations.service.ts
Comment (Heba-WebDev) [inline src/modules/organisations/organisations.service.ts:1]: addUser method is doing so much. Let it do one thing only and extract the rest into helper methods.

**Review by Heba-WebDev — APPROVED** [review]

**Review by davidolanrewaju — APPROVED** [review]

**Review by davidolanrewaju — APPROVED** [review]

**Comment by davidolanrewaju** [comment]
Please fix your workflow, pull the latest changes to the repo run your install and push back

**Review by davidolanrewaju — DISMISSED** [review]

**Review by Heba-WebDev — DISMISSED** [review]

File: src/modules/organisations/dto/add-user-dto.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { IsString } from 'class-validator';
+
+export class AddUserDto {
```
Comment (Homoakin619) [inline src/modules/organisations/dto/add-user-dto.ts:3]: please change name to AddUserToOrganizationDto

**Review by davidolanrewaju — DISMISSED** [review]

File: src/modules/organisations/entities/org-members.entity.ts
Comment (Heba-WebDev) [inline src/modules/organisations/entities/org-members.entity.ts:1]: Please add nullable to each column, nullable: true/false

**Review by Heba-WebDev — DISMISSED** [review]

**Review by Cobidex — DISMISSED** [review]
